### PR TITLE
fix/duplicated grouped investigation

### PR DESCRIPTION
**Pull request by y-golde**
fixed in issue where the table would show duplicated in the grouping table, fixes [1423](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/1423/)

fixed typo

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx [hunk 1]
```diff
@@ -25,21 +25,26 @@ const GroupedInvestigations: React.FC<Props> = (props: Props) => {
...
+            const investigationsToGroupIds = investigationsToGroup.map(investigation => investigation.epidemiologyNumber);
             const groupedInvestigations = allGroupedInvestigations.get(groupedInvestigation.groupId)
-                ?.filter((investigation: InvestigationTableRow) => investigation.epidemiologyNumber !== groupedInvestigation.epidemiologyNumber);
-            const investigationsToDisplayUpdateMode = groupedInvestigations && invetigationsToGroup.concat(groupedInvestigations);
+                ?.filter((investigation: InvestigationTableRow) => {
+                    return investigation.epidemiologyNumber !== groupedInvestigation.epidemiologyNumber 
+                            && investigationsToGroupIds.indexOf(investigation.epidemiologyNumber) === -1; 
+                });
+            const investigationsToDisplayUpdateMode = groupedInvestigations && investigationsToGroup.concat(groupedInvestigations);
+            console.log('aaa' , investigationsToDisplayUpdateMode);
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx:42]: delete

**Review by ItayBenMoshe — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx [hunk 2]
```diff
@@ -25,21 +25,26 @@ const GroupedInvestigations: React.FC<Props> = (props: Props) => {
...
     
     const groupedInvestigation = useMemo(() => {
-        return invetigationsToGroup.find((investigation: InvestigationTableRow) => investigation.groupId);
-    }, [invetigationsToGroup]);
+        return investigationsToGroup.find((investigation: InvestigationTableRow) => investigation.groupId);
+    }, [investigationsToGroup]);
     
     const title = groupedInvestigation ? 'הוספת חקירות לקיבוץ' : 'קיבוץ חקירות'
 
     const investigationsToDisplay: InvestigationTableRow[] = useMemo(() => {
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx:33]: `const investigationsToDisplay: InvestigationTableRow[] = useMemo(() => {
        if (groupedInvestigation) {
            const groupedInvestigations = allGroupedInvestigations.get(groupedInvestigation.groupId);
            const investigationsWithoutGroupIds = invetigationsToGroup.filter((investigation: InvestigationTableRow) => !investigation.groupId);
            const investigationsToDisplayUpdateMode = groupedInvestigations && groupedInvestigations.concat(investigationsWithoutGroupIds);
            return investigationsToDisplayUpdateMode ? investigationsToDisplayUpdateMode: [];
        } else {
            return invetigationsToGroup;
        }
    }, [groupedInvestigation, invetigationsToGroup, allGroupedInvestigations])`

Comment (ItayBenMoshe) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx:33]: try this.

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx (hunk 1 above)
Comment (y-golde) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx]: oops, fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1189/commits/acab238b857c52ecaecc243acb8c14c1e381578d)

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx (hunk 2 above)
Comment (y-golde) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx:33]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1189/commits/1b0e1767764ef232c96bf9fcb671393642efffe4)

**Review by ItayBenMoshe — APPROVED** [review]

**Review by Noam6421 — APPROVED** [review]

**Pull request merged**